pkgs/development/python-modules/mkdocs-rss-plugin/default.nix

```
@@ -0,0 +1,26 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+}:
+
+buildPythonPackage {
+  pname = "mkdocs-rss-plugin";
+  version = "1.8.0";
+
+  src = fetchFromGitHub {
+    owner = "Guts";
+    repo = "mkdocs-rss-plugin";
+    rev = "1.8.0";
+    sha256 = "sha256-rCz1Uk5uqIsnIWw0b1oBsjAO6aK/tpVgqAX/8dVnAGw=";
```

SRI hashe should be using `hash`.

```suggestion
    hash = "sha256-rCz1Uk5uqIsnIWw0b1oBsjAO6aK/tpVgqAX/8dVnAGw=";
```